src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -308,7 +308,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             }
                             else
                             {
-                                instrument = value;
+                                instrument = RemoveQuotesAndSlashes(value);
```

Comment:
This should be done before we check value for null or empty. Please add a test for this scenario.